### PR TITLE
Add a precision about the API layers

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/api.rst [hunk 1]
```diff
@@ -34,3 +34,9 @@ In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js`` you can custom
    config.queryableLayers = [<layer a>, <layer b>, ...];
 
 The API help is in the folder ``geoportal/<package>_geoportal/static/apihelp/``.
+
+To be able to have a layer in the api, the layer should be in the themes::
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: Should this read:
"In order for a layer to be accessible via API, the layer must be in a group"

(below we are talking about "group", not "theme")

Comment (sbrunner) [reply doc/integrator/api.rst]: in the themes (imply that the group should already be in a visible theme) or do you have a better proposition?

File: doc/integrator/api.rst [hunk 2]
```diff
@@ -34,3 +34,10 @@ In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js`` you can custom
    config.queryableLayers = [<layer a>, <layer b>, ...];
 
 The API help is in the folder ``geoportal/<package>_geoportal/static/apihelp/``.
+
+In order for a layer to be accessible via API, the layer must be in the themes::
+
+ * Add the layer
+ * Select the api interface
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: API

File: doc/integrator/api.rst [hunk 3]
```diff
@@ -34,3 +34,10 @@ In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js`` you can custom
...
+ * Add the layer to a group (the group should already be visible by the api,
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: Suggestion:

* Add the layer to a group which is visible in the API.

Note: a group is visible in the API if it is in a theme whose API configuration option is activated.

File: doc/integrator/api.rst [hunk 4]
```diff
@@ -34,3 +34,10 @@ In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js`` you can custom
    config.queryableLayers = [<layer a>, <layer b>, ...];
 
 The API help is in the folder ``geoportal/<package>_geoportal/static/apihelp/``.
+
+In order for a layer to be accessible via API, the layer must be in the themes::
```
Comment (jwkaltz) [inline doc/integrator/api.rst]: Suggestion:
, the layer must be in the themes::
->
, proceed as follows::

File: doc/integrator/api.rst (hunk 3 above)
Comment (sbrunner) [reply doc/integrator/api.rst]: Adapted and updated, what do you think?

Comment (jwkaltz) [reply doc/integrator/api.rst]: I like this except that the "the" should be removed in 
"whose the 'api' interface"

Comment (sbrunner) [reply doc/integrator/api.rst]: OK, updated :-), thanks

**Pull request merged**